src/main/java/step3/view/InputView.java

```diff
@@ -0,0 +1,37 @@
+package step3.view;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+import step3.common.exception.InvalidParamException;
+import step3.dto.WinnerLottoNumbersDto;
+
+public class InputView {
+    private static final String ONLY_NUMBER = "숫자만 입력 해주세요.";
+
+    private static final Scanner sc = new Scanner(System.in);
+    private final ResultView resultView = new ResultView();
+
+    public int[] readLineToArray() {
+        return Stream.of(sc.next().split(",")).mapToInt(Integer::parseInt).toArray();
```

Comment:
스트림은 행위 단위로 내려쓰는 것이 가독성과 디버깅시 더 좋습니다.

```suggestion
        return Stream.of(sc.next().split(","))
        .mapToInt(Integer::parseInt)
        .toArray();
```

Reply:
다른 부분에도 가독성에 대한 부분 검토해보겠습니다!